contact-us.html

```diff
@@ -120,33 +151,40 @@ <h2 class="get-in-title">
                     Get in touch
                 </h2>
                 <p class="get-in-text">Browse through the most freuently asked questions</p>
-                <form method="get">
-                    <input class="get-in-input"
+                <form action="#" method="get" class="contact-us-form">
+                    <span class="contact-info">
+                        <input class="get-in-input name"
                         type="text" 
```

Comment:
All elements inside form (except button) should have labels according to acceptance criteria